slurm-stress/fractal-slurm/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:22.04
+
+ARG SLURM_VERSION=23.02.7
+ARG SLURM_ROOT=/opt/slurm-${SLURM_VERSION}
+ARG SLURM_CONFDIR=${SLURM_ROOT}/etc
+
+RUN apt update -y
+RUN apt install -y munge build-essential mariadb-server wget bsdmainutils
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN mkdir /run/munge
+RUN chmod 777 /run/munge
+RUN chmod +t /run/munge
+
+RUN apt install -y slurmd slurm-client slurmctld
+RUN apt install -y python3 python3-pip
+RUN useradd -m admin -s /usr/bin/bash -d /home/admin && echo "admin:admin" | chpasswd && adduser admin sudo && echo "admin     ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+RUN useradd --uid 2001 -m test01 -s /usr/bin/bash -d /home/test01 && echo "test01:test01" | chpasswd
+
+RUN mkdir -p /var/log/slurm
+RUN mkdir -p /var/spool/slurmctld && chown slurm /var/spool/slurmctld && chmod u+rwx /var/spool/slurmctld
+RUN mkdir -p /var/spool/slurmd    && chown slurm /var/spool/slurmd    && chmod u+rwx /var/spool/slurmd
+
+
+COPY install_slurm.sh .
+RUN apt install -y curl
+RUN  ./install_slurm.sh ${SLURM_VERSION} ${SLURM_ROOT} ${SLURM_CONFDIR} --enable-multiple-slurmd
+
+COPY slurm.conf.in /etc/slurm/
+COPY cgroup.conf /etc/slurm/
+COPY docker-entrypoint.sh /etc/slurm/
+
+ADD sudoers /etc/sudoers
+COPY fractal_tasks_mock-0.0.1-py3-none-any.whl ./
+COPY slurm_config.json ./
+COPY run_server.sh ./
+COPY monitor_threads.sh ./
+COPY .fractal_server.env ./
+RUN apt-get -y install gcc postgresql postgresql-contrib libpq-dev
+RUN pip3 install --upgrade pip setuptools wheel
+RUN pip3 install "fractal-server[gunicorn,postgres]==2.0.6"
```

Comment:
use 2.2.0a1, so that we can switch to psycopg3

The new postgres adapter by @ychiucco (fully based on psycopg3, rather than psycopg2+asyncpg) is available starting from `fractal-server=2.2.0a1`. To use it, one should
1. Switch from `pip install fractal-server[postgres]` to `pip install fractal-server[postgres-psycopg-binary]`
2. Switch from `DB_ENGINE=postgres` to `DB_ENGINE=postgres-psycopg` in the `fractal-server` configuration variables